### PR TITLE
TXT values specification

### PR DESCRIPTION
**Pull request by eduardo-cunha-bose**
Now it is possible to specify values for TXT DNS resource records.
In a service configuration, we can add the txt_payload key which which
should be a list of strings containing a key and value separated
by a '='.

### CODE REVIEW

**Comment by eduardo-cunha-bose** [comment]
cc @fhunleth @jjcarstens

File: README.md [hunk 1]
```diff
@@ -109,8 +109,10 @@ The `services` section lists the services that the host offers, such as
 providing an HTTP server. You must supply the `protocol`, `transport` and `port`
 values for each service. You may also specify `weight` and/or `host`.  They each
 default to a zero value. Please consult the RFC for an explanation of these
-values. Services can be configured in `config.exs` as shown above, or at
-runtime:
+values. You can also specify `payload` which is used to define the data in a TXT
```
Comment (fhunleth) [inline README.md:112]: I'm not familiar with the proper naming of this, but I'm wondering if there will be other payloads in the future. Like maybe this should be `txt` or `txt_payload`. (If you don't know, I'll skim through the RFC to see if it has any naming guidance.)

**Comment by fhunleth** [comment]
@eduardo-cunha-bose Thanks for resubmitting this here! I was just thinking about this the other day. I think that it looks good with the only caveat being the parameter name question.

File: README.md (hunk 1 above)
Comment (eduardo-cunha-bose) [reply README.md:112]: I also don't know what could be an appropriate name, `txt_payload` seems a really nice suggestion.

Comment (jjcarstens) [reply README.md:112]: 👍 for `txt_payload`

Comment (fhunleth) [reply README.md:112]: I don't see anything better in the RFC. Sounds like we're going with `txt_payload`. @eduardo-cunha-bose, could you update the PR?

**Comment by eduardo-cunha-bose** [comment]
The change is made from payload to txt_payload.

By the way, the mix.exs file should be updated so that the hex package points to this repository instead of the original one.
Thats why I opened initially the PR on the other repository, because the documentation and hex package are pointing to the other repository.

**Comment by fhunleth** [comment]
Thanks x2. I'm going to update the URLs shortly.

**Pull request merged**